tests/benchdnn/rnn/rnn.cpp

```diff
@@ -825,13 +836,24 @@ void skip_unimplemented_prb(const prb_t *prb_, res_t *res) {
             res->reason = skip_reason::case_not_supported;
             return;
         }
-        if (is_cpu()
-                && (prb.tag[0] != tag::abx || prb.tag[1] != tag::any
-                        || prb.tag[2] != tag::abx)) {
-            res->state = SKIPPED;
-            res->reason = skip_reason::case_not_supported;
-            return;
+
+        if (is_cpu()) {
+            const auto src_tag
+                    = normalize_tag(prb.tag[0], prb.ndims(SRC_LAYER));
+            const auto wei_tag
+                    = normalize_tag(prb.tag[1], prb.ndims(WEIGHTS_LAYER));
+            const auto dst_tag
+                    = normalize_tag(prb.tag[2], prb.ndims(DST_LAYER));
+
+            const bool tags_not_ok = src_tag != "abc" || wei_tag != tag::any
```

Comment:
Nit: I'd invert the condition for it to be positive (`tags_ok`), because it's usually easier to append the valid configuration through a dedicated variable than to fit it into negation condition.

Reply:
The reason why is used `not` is because there are already `is_f16_not_ok` and `is_acl_f16_not_ok` above so I just followed the style. I don't have a strong preference and can drop `not` if you insist.